app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Admin
+  class OrdersController < ApplicationController
+    def index
+      render :index, locals: { orders: Order.all }
```

Comment:
it looks like a n+1 to me :)

Reply:
I used the bullet gem to check the app and removed all n+1